.github/workflows/eks-ci.yml

```diff
@@ -191,7 +197,7 @@ jobs:
           kubectl exec $RESULTS_POD -- sh -c 'rm -rf /reports/*'
 
       - name: Upload Test Results
-        if: success()
+        if: always()
```

Comment:
what happens when the test fails?

Reply:
This action uploads test results. There are two type of errors here:

1. Action failure as a test results failure - This is the correct case and upload results are required in order to be able to track them in Allure report.
2. Action error due to unforeseen circumstances - In this case the action will try to upload the results and will fail.